config.yml.erb

```diff
@@ -630,3 +630,7 @@ openai_evaluate_rubric_org_id:
 
 # AI Proxy for OpenAI API
 ai_proxy_origin: https://aiproxy-test.code.org
+
+# Statsig keys
```

Comment:
Will Pegasus pick up `production` keys? Or is there an additional configuration I need to add?

Reply:
Yes. The `CDO` singleton is initialized by both Pegasus and Dashboard web application servers (and also any code the requires `/deployment.rb` such as cron jobs).